theme/less/override.less

```diff
@@ -11,6 +11,7 @@ ul {
 
 a:not([href]) {
     cursor: pointer;
+    color: unset;
```

Comment:
Where is it needed?

Reply:
The DSFR add a new default color to link without href.

In udata-front, links used with @click doesn't have a href. This is bad for accessibility reason so I remove them  when I see them but it's a pattern in our codebase.